### DISCUSSION TITLE
Issue after '.' (period), tts. voice is changing

### DISCUSSION THREAD

**Discussion by Chillae** [neonbjb/tortoise-tts#69] [General]
Hello everyone, great project!
I have run into an issue, sometimes after a . (period) or comma the tts switches voice. For example, I have set my voice to be jlaw, but after a . (period) it switches to a voice I can only describe as joe rogan. Which is strange, because he doesnt seem to be in the voices folder. 

A work around is to not use periods, but this makes the voice sound a bit strange, without any stops. 

Is there another way to add pauses to the output speech?

**Comment by neonbjb** [comment]
Someone else pointed this out to me in a different context a couple of days ago. Rogan's podcast was part of the training dataset and it would appear that I did a poor job splitting it up such that there must be some clips where both him and a target voice were talking in the same clip. This must have translated to the model learning that sometimes a period means "inject Joe Rogan talking".

Unfortunately there's not much I can do now, other than re-train the model. If I do give that a go someday, I'll certain do another filtering pass, or just remove the Rogan podcast altogether..

**Reply by Chillae** [reply]
Haha, Yes I thought the model might have been trained on joe rogan, hence his voice being there. Made me laugh when I heard it. 

Is there any way to create a pause mid sentence? I guess I could re-run the sections that have changed and then edit them together, if I wanted a long sentence.

**Comment by neonbjb** [comment]
Is it really that common? I don't recall seeing this at all when I was testing the jlaw voice when I originally released Tortoise. I would have assumed this happens <10% of the time and the answer would be to "just re-render that clip".

I'm guessing you already tried using other punctuation like semicolons or dashes?

**Comment by neonbjb** [comment]
I wonder if this is caused by CLVP2, hence the recent reports..

If you have time and can easily reproduce this, would you mind reverting to fda51308196c9baa4e0552fd85c312fefd512f88 and re-trying your tests?

**Comment by scott-vector** [comment]
This doesn't work when moving back to CLVP. When we run sentences with . ; - we get a different sounding voice for each sentence, most of the time. When we remove these separators, we get a steady, single voice.
